lib/diagnostic.js

```diff
@@ -13,7 +13,8 @@ const add = function(num1, num2){
 add();
 
 // Question 1:  What is the return value of `add();`?
-let answer1 = null; // replace null with your response
+let answer1 = 3; // or its 1 I don't know if num1 technically changes or not
```

Comment:
`num1` and `num2` are `undefined` in the context of `add()`, so `num1 + num2` evaluates to `undefined + undefined`, which is `NaN`.